doc/modules/preprocessing.rst

```diff
@@ -426,6 +426,47 @@ features, one has to explicitly set ``n_values``. For example,
 See :ref:`dict_feature_extraction` for categorical features that are represented
 as a dict, not as integers.
 
+.. _discretization:
+
+Discretization
+==============
+
+`Discretization <https://en.wikipedia.org/wiki/Discretization_of_continuous_features>`_
+(otherwise known as quantization or binning) provides a way to partition continuous
+features into discrete values. Certain datasets with many continuous features
```

Comment:
Not sure about this statement on "certain datasets with many continuous features". I suspect that it is applicable to datasets with few continuous features too.